### ISSUE TITLE
Frozen Depth roguelike not working

### ISSUE THREAD

**Issue by rumly111** [ptitSeb/box64#1942]
Frozen Depth roguelike game didn't start. Both ncurses and ncursesw versions.
I don't remember the exact error message, but something about the terminal not having enough rows and columns for the game to run. 

I ran it on my Pi3, which is now dusting on the shelf. If it's really needed, I can boot it up and post the full error log, but I think downloading FrozenDepth and checking it out is much faster.

**Comment by ptitSeb** [comment]
That doesn sound like  a box issue, but more like n application isue: it wants a bigger terminal. Enlarge the terminal window?

**Comment by rumly111** [comment]
nope, the terminal was big enough

**Comment by ptitSeb** [comment]
Ah, ok. I'll try to check later.

**Comment by ptitSeb** [comment]
Yeah, looks like the wrapping of ncurses is lacking. It might work better with an emulated ncurses. I'll add the issue to my TODO...

**Comment by rumly111** [comment]
thank you!

**Comment by LukeShortCloud** [comment]
@rumly111 Can you try to:

1. First create a [bundle of x86 libs](https://github.com/ptitSeb/box64/blob/main/docs/BUNDLE-X86-LIBS.md) (I just added `ncurses` library files).
2. Then run you program again with `BOX64_EMULATED_LIBS=libncurses.so:libncurses.so.6:libncursesw.so:libncursesw.so.6` (set either via an environment variable or RC file).
    - https://github.com/ptitSeb/box64/blob/v0.3.6/docs/USAGE.md#box64_emulated_libs

That'll use emulation instead of wrapping which might work better in this case.
